testinput_tier_1/inputs/gfs_c12/mem000

```diff
@@ -0,0 +1 @@
+/gpfsm/dnb05/projects/p61/rtodling/JEDI1/2024/Sep11/jedi/build-intel-release/fv3-jedi/test/Data/inputs/gfs_c12/bkg
```

Comment:
This looks like a symlink to a very specific directory on some HPC system. Can it be made a relative path like "../bkg" or similar?

Reply:
@rtodling ping that I think we have a problem to address here.

Reply:
@rtodling you probably added this file by mistake, ca n you remove it? Yannick fixed the issue where this file was being added very recently